cypress/integration/settings/circulation/fixed-due-date-schedules/schedule-does-not-cover-the-test-date.spec.js

```diff
@@ -0,0 +1,177 @@
+                    const noticePolicyId = Cypress.env(CY_ENV.NOTICE_POLICY)[0].id;
+                    const overdueFinePolicyId = Cypress.env(CY_ENV.OVERDUE_FINE_POLICY)[0].id;
+                    const lostItemFeesPolicyId = Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY)[0].id;
+                    const newRule = `\nm ${Cypress.env(CY_ENV.MATERIAL_TYPES)[0].id}: l ${loanPolicyId} r ${requestPolicyId} n ${noticePolicyId} o ${overdueFinePolicyId} i ${lostItemFeesPolicyId}`;
+
+                    cy.updateCirculationRules({
+                      rulesAsText: rulesDefaultString + newRule,
+                    });
+                  })
+                  .then(() => {
+                    cy.createItemCheckout({
+                      servicePointId: Cypress.env(CY_ENV.SERVICE_POINTS)[0].id,
+                      itemBarcode: ITEM_BARCODE,
+                      userBarcode: USER_BARCODE,
+                      loanDate: moment.utc(),
+                    });
+                  });
+              });
+          });
+      });
+  });
+
+  after(() => {
+    cy.createItemCheckinApi({
+      itemBarcode: ITEM_BARCODE,
+      servicePointId: Cypress.env(CY_ENV.SERVICE_POINTS)[0].id,
+      checkInDate: moment.utc().format(),
+    })
+      .then(() => {
+        cy.deleteUser(userId);
+        cy.getInstance({ limit: 1, expandAll: true, query: `"items.barcode"=="${ITEM_BARCODE}"` })
+          .then((instance) => {
+            cy.deleteItem(instance.items[0].id);
+            cy.deleteHoldingRecord(instance.holdings[0].id);
+            cy.deleteInstanceApi(instance.id);
+          });
+        cy.updateCirculationRules({
+          rulesAsText: rulesDefaultString,
+        });
+        cy.deleteLoanPolicy(Cypress.env(CY_ENV.LOAN_POLICY).id)
+          .then(() => {
+            cy.deleteFixedDueDateSchedule(Cypress.env(CY_ENV.FIXED_DUE_DATE_SCHEDULE).id);
+          });
+      });
+  });
+
+  it('C641: renewing item using a fixed due date loan profile where the fixed due date schedule date range does not cover the test date',
+    { tags: [TestTypes.smoke] },
+    () => {
+      cy.visit(SettingsMenu.circulationFixedDueDateSchedulesPath);
+      fixedDueDateSchedules.editSchedule(Cypress.env(CY_ENV.FIXED_DUE_DATE_SCHEDULE).name, {
+        description: Cypress.env(CY_ENV.FIXED_DUE_DATE_SCHEDULE).description,
+        schedules:[{
+          from: fromDate.format('YYYY/MM/DD'),
+          to: newToDate.format('YYYY/MM/DD'),
+          due: dueDate.format('YYYY/MM/DD'),
+        }],
+      });
+      cy.contains(`${newToDate.format('M/D/YYYY')}`).should('be.visible');
+      cy.visit(topMenu.checkOutPath);
+      checkout.checkUserOpenLoans(Cypress.env(CY_ENV.USER));
+      cy.contains(`${Cypress.env(CY_ENV.LOAN_POLICY).name}`).should('be.visible');
+      loans.renewLoan();
+      cy.contains('renewal date falls outside of date ranges in fixed loan policy').should('be.visible');
```

Comment:
wrap to method with meaning name into appropriate fragment

Reply:
Done